implementations/nestjs-code-first/.gitignore

```diff
@@ -0,0 +1,398 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
this is a pretty large `.gitignore` file -> should we trim it down to be specific to this impl? or update root ignore file with additional configs (if needed)?

Reply:
For sure - this is the `.gitignore` Nest's bootstrap project includes. Happy to trim it down!

Reply:
I just killed it completely - we don't need any of it. I'm not sure why all of that is in their bootstrap project (are folks really using IntelliJ with NestJS? 🤷).